dccrg.hpp

```diff
@@ -9734,7 +9747,7 @@ template <
 					this->neighborhood_of,
 					2 * this->max_ref_lvl_diff,
 					true
-				);
+				);	// todo PRETTY SURE THIS SHOULDN'T BE HERE!
```

Comment:
Please explain

Reply:
I'm _fairly_ certain that bit should just be using cached neighbors instead of determining them all over again. That isn't dealing with face neighbor caching though, so kind of out of scope for the PR